### DISCUSSION TITLE
Starting from an existing install

### DISCUSSION THREAD

**Discussion by dpeet** [meisnate12/Plex-Meta-Manager#117] [Q&A]
What is the recommended way to start building up a yaml file?  Is there a way to export your overrides from a current plex installation?

Thanks in advance! :D

**Comment by dpeet** [comment]
@meisnate12 do you have any wisdom here?

**Reply by meisnate12** [reply]
I don't know how I didn't see this earlier I'm out now but if I don't get back to you tonight comment again tomorrow to remind me

**Comment by meisnate12** [comment]
so currently PMM doesn't have a way to detect changes so it cant pull the metadata for you. This is a planned feature but its a big undertaking so i haven't started that yet. What i did in the short term was use [Export Tools](https://forums.plex.tv/t/webtools-ng/598539) to download changes in my db and then i manually added them into my metadata config. It's not a super easy process but one day i hope to have it happen much more easily
